firestore/pom.xml

```diff
@@ -59,8 +69,9 @@
       <version>4.13-beta-3</version>
       <scope>test</scope>
     </dependency>
+    <!-- [START fs-maven] -->
```

Comment:
Remove these and check the rest of your changes for these weird tags, looks like a merge error.